spec/spec_helper.rb

```diff
@@ -12,6 +9,7 @@
   add_filter '/spec/'
 end
 
+ENV['ROBOT_ENVIRONMENT'] = 'test'
```

Comment:
do we still need this, if we got rid of the config/environments?

Reply:
Yes, this is also used to determine which `config/settings/*.yml` file to load.

See https://github.com/sul-dlss/lyber-core/blob/main/lib/lyber_core/boot.rb#L40